azure/hostedservice/entities.go

```diff
@@ -1,9 +1,16 @@
-package hostedServiceClient
+package hostedservice
 
 import (
 	"encoding/xml"
+
+	"github.com/MSOpenTech/azure-sdk-for-go/azure"
 )
 
+//HostedServiceClient is used to manage operations on Azure Hosted Services
+type HostedServiceClient struct {
+	client *azure.Client
```

Comment:
I considered this, but it would probably be better to be consistent and there's no guarantee there won't be some other client state we want in here.